docs/source/getting_started.rst

```diff
@@ -218,16 +218,16 @@ slower):
 TIC calculation
 ^^^^^^^^^^^^^^^
 
-Here, we will apply what we have learned to calculate the total ion current. 
+Here, we will apply what we have learned to calculate the total ion chromatogram (TIC). The TIC is a 
+chromatogram created by summing up intensities of all mass spectral peaks belonging to the same ms run.
 While some mzML files already contain a pre-computed TIC, we will show you how to calculate the TIC for MS1
 and MS2 spectra separately (generally only MS1 TIC is provided).
 
-With this information, we can write a function that calculates the total ion
-current (TIC) for a given ms level: 
+With this information, we can write a function that calculates the TIC for a given ms level: 
 
 .. code-block:: python
 
-    # Calculates total ion current of an LC-MS/MS experiment
+    # Calculates total ion chromatogram of an LC-MS/MS experiment
```

Comment:
this calculates the current, right? not the chromatogram

Reply:
Yes, you are right - I misunderstood. 

A TIC (Total Ion Chromatogram) is a chromatogram created by summing up intensities of all mass spectral peaks belonging to the same scan. So basically at a specific retention time. 

The total ion current (TIC) chromatogram represents the summed intensity across the entire range of masses being detected at every point in the analysis.